master/static/migrations/20230928101029_rename-last-login-column-in-user-table.tx.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE IF EXISTS public.users RENAME COLUMN last_auth_at TO last_login;
```

Comment:
we should really expect users to exist, I don't think we need the IF EXISTS

Reply:
ok. will remove it